pages/index.js

```diff
@@ -34,104 +34,29 @@ import {
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-      <Box as='main' h={'full'} w={'full'}>
-        {/* Show a circular loader from chakra ui */}
-        <Box as='header' w={'full'} p={4} borderBottomWidth={1} borderBottomColor={'gray.200'} bg={'gray.900'} mb={12}>
-          <Box py={'16'}>
-            <Heading color={'white'} textAlign={'center'}>
-              Explore Status updates for Summerdale community
-            </Heading>
-            <Text size={'xl'} color={'gray.300'} textAlign={'center'}>
-              A status page to help Summerdale Park home owners stay up to date with their homes.
-            </Text>
-          </Box>
-          <Stack spacing={4} mb={-8} maxW={672} marginInline="auto" alignItems={'center'}>
-            <InputGroup >
-              <InputLeftElement
-                flex={1}
-                pointerEvents='none'
-              >
-                <SearchIcon color="gray.300" fontSize={'xl'} />
-              </InputLeftElement>
-              <Input size="lg" bg={'white'} type='tel' placeholder='Search for a lot number' />
-              <InputRightElement width='4.5rem'>
-                <Kbd>cmd</Kbd>
-                <Kbd>k</Kbd>
-              </InputRightElement>
-            </InputGroup>
-          </Stack>
-        </Box>
-        <Box as='section' w={'full'} p={4}>
-          <Container maxW='container.xl'>
-            {isLoading && data.length === 0 ? <CircularProgress isIndeterminate /> : <Card variant={'outline'}>
+function ResultsTable({data}) {
+  return <Card variant={'outline'}>
               <CardBody>
                 <TableContainer>
                   <Table variant='simple'>
                     <Thead>
                       <Tr>
-                        <Th>Permit #</Th>
                         <Th>Lot #</Th>
-                        <Th>Parcel #</Th>
                         <Th>Address</Th>
-                        <Th>Contractor phone</Th>
+                        <Th>Status?</Th>
                       </Tr>
                     </Thead>
                     <Tbody>
                       {data.map((item) => {
                         return (
-                          <Tr key={item.permit_number}>
-                            <Td>{item.permit_number}</Td>
-                            <Td>
-                              {item.parcel_number.slice(-5, -1)}
-                              {/*19374 BANFIELD PL */}
-                            </Td>
-                            <Td>{item.parcel_number}</Td>
-                            <Td>{item.permit_address}</Td>
-                            <Td>{item.contractor_phone_number}</Td>
-                            <Td isNumeric>{item.zip}</Td>
+                          <Tr key={item.number}>
+                            <Td>{item.number}</Td>
+                            <Td>{item.address}</Td>
+                            <Td>?</Td>
```

Comment:
only showing lot number, address and hopefully in the future we'll be able to figure out how to give a "status" of the lot like "complete" or pre-drywall etc...
I'm not sure how to do that though.. maybe Bruno will know

Reply:
Sounds good! Added it here: https://github.com/agnelnieves/summerdale-permits/issues/6